core-web/libs/dotcms-models/src/lib/dot-workflow.model.ts

```diff
@@ -3,7 +3,7 @@ export interface DotCMSWorkflow {
     creationDate: Date;
     defaultScheme: boolean;
     description: string;
-    entryActionId: string;
+    entryActionId: string | null;
```

Comment:
why not `entryActionId?`

Reply:
For now is neccesary | null, entryActionId? makes it possible undefined, but in test this property is send as null, probably the backend needs this as null